app/containers/MessageBox/RightButtons.ios.js

```diff
@@ -1,26 +1,19 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import { SendButton, AudioButton } from './buttons';
+import { SendButton } from './buttons';
 
-const RightButtons = React.memo(({
-	theme, showSend, submit, recordAudioMessage, recordAudioMessageEnabled
-}) => {
+const RightButtons = React.memo(({ theme, showSend, submit }) => {
 	if (showSend) {
 		return <SendButton theme={theme} onPress={submit} />;
 	}
-	if (recordAudioMessageEnabled) {
-		return <AudioButton theme={theme} onPress={recordAudioMessage} />;
-	}
```

Comment:
Why? This was already at the right of the screen.

Reply:
We don't need the `AudioButton` now. The `RecordAudio` component itself holds the UI and the Gestures

Reply:
sure